features/pom.xml

```diff
@@ -36,6 +36,7 @@
     </distributionManagement>
 
     <modules>
+	<module>APIM</module>  
```

Comment:
It is correctly formatted in the pom file. Problem was space charters should use instead of tabs. Thanks